packages/repo-tools/src/commands/index.ts

```diff
@@ -60,6 +64,16 @@ function registerSchemaCommand(program: Command) {
     .command('init <paths...>')
     .description('Creates any config needed for the test command.')
     .action(lazy(() => import('./openapi/test/init').then(m => m.default)));
+
+  openApiCommand
+    .command('generate-client')
```

Comment:
Yep think this makes sense. We could potentially consider having the other one be `generate-server` and ofc if possible it might be nice to consolidate into a single `generate` command